CAE/CorporateEvents/CorporateActions.rdf

```diff
@@ -182,31 +237,44 @@
 		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;VoluntaryCorporateAction"/>
 		<rdfs:label xml:lang="en">disclosure action</rdfs:label>
 		<skos:definition xml:lang="en">corporate action involving a requirement for holders or beneficial owners to disclose their name, location and holdings of any issue to the issuer</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">This event only occurs with the characteristic VOLU, otherwise it is part of the SRD II.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;DivestitureAction">
+		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;MandatoryCorporateAction"/>
+		<rdfs:label xml:lang="en">divestiture action</rdfs:label>
+		<skos:definition xml:lang="en">corporate action involving the sales of subsidiary business interests or investments</skos:definition>
+		<skos:note xml:lang="en">Divestiture is an action or process involving the sale, spinoff, or liquidation of business assets, including product lines, services, subsidiaries, business property, or even an entire business. Companies may pursue a divestment strategy to refocus on their core business, in response to the operating environment in their industry, to raise capital, or to release underperforming assets.</skos:note>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-cae-ce-act;IncomeOrientedClassifier">
+		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;ActionClassifier"/>
+		<rdfs:label xml:lang="en">income-oriented classifier</rdfs:label>
+		<skos:definition xml:lang="en">classifier of corporate actions that impacts income to shareholders</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">Cash dividends are a classic example where a public company declares a dividend to be paid on each outstanding share. Bonus is another case where the shareholder is rewarded. In a stricter sense, the bonus issue should not impact the share price but in reality, in rare cases, it does and results in an overall increase in value.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-cae-ce-act;LegalFormChange">
 		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;CorporateInformationAction"/>
 		<rdfs:label xml:lang="en">legal form change</rdfs:label>
 		<skos:definition xml:lang="en">corporate action that informs stakeholders of a modification of the legal form of the organization</skos:definition>
```

Comment:
This doesn't change the legal form with a jurisdiction, it lets people know that the form has changed

Reply:
I've revised the text to say that it indicates a change in the form rather than is information about the change, and revised it to be a business strategy action as opposed to an information change